scope & closures/ch5.md

```diff
@@ -43,7 +43,7 @@ function foo() {
 foo();
 ```
 
-This code should look familiar from our discussions of Nested Scope. Function `bar()` has *access* to the variable `a` in the outer enclosing scope because of lexical scope look-up rules (in this case, it's an RHS reference look-up).
+Esse código pode parecer familiar das nossas discussões sobre Escopo Aninhado. A função `bar()` tem *acesso* à variável `a` do escopo externo ao redor por causa das regras de consulta ao escopo léxico (neste caso, é uma referência RHS).
```

Comment:
Fiquei com dúvida na parte *`(...)in the outer enclosing scope`*. Eu penso que a palavra *`enclosing`* está com o sentido de `fechamento`. Sugestão `no escopo de fechamento externo` ou algo nesse sentido.

Segue a função citada.

```JavaScript
function foo() {
	var a = 2;

	function bar() {
		console.log( a ); // 2
	}

	bar();
}

foo();
```
www.wordreference.com/definition/enclosing?s=enclosing